chcemvediet/apps/anonymization/migrations/0001_initial.py

```diff
@@ -19,7 +19,7 @@ class Migration(migrations.Migration):
                 ('file', models.FileField(help_text="Empty filename if normalization failed or normalization didn't create any file.", max_length=255, blank=True, upload_to='attachment_normalizations')),
```

Comment:
misc/anonymization.odt : V dokumentacii je pre model AttachmentNormalization uvedene 
```
file : May be NULL.
```
Prepisem to a do poznamky uvediem nieco v tom zmysle ze moze byt `<FieldFile: None>` ?

Reply:
V `anonymization.odt` je veta "May be NULL" samozrejme zle. Oprav to na "May NOT be NULL". A potom v dalsom riadku sa pise "Empty filename if..."

Reply:
:ok: